earthpy/spatial.py

```diff
@@ -56,7 +56,7 @@ def normalized_diff(b1, b2):
     Returns
     ----------
     n_diff : ndarray with the same shape as inputs
-        The element-wise result of (b2-b1) / (b2+b1) with all nan values
+        The element-wise result of (b2-b1) / (b2+b1) with all nan and inf values
         masked.
```

Comment:
i think this is where we could have another statement here that says something like.:

 All values == Inf will  be set to np.nan.  (or something like that just to be explicit!

Reply:
The docstring has been updated to explicitly state that inf values are set to nan.